File: Back/src/ProEvents.Domain/Event.cs
using System.ComponentModel.DataAnnotations.Schema;
using ProEvents.Domain.Identity;

namespace ProEvents.Domain
{
    //[Table("")]
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? EventDate { get; set; }
        public string Theme { get; set; }
        public int QtdPeople { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<Lot> Lots { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<EventSpeaker> EventsSpeakers { get; set; }
    }
}

File: Front/ProEvents-App/src/app/models/Event.ts
import { SocialMedia } from './SocialMedia';
import { Lot } from './Lot';
import { Speaker } from './Speaker';

export interface Event {
  id: number;
  local: string;
  eventDate?: Date
  theme: string;
  qtdPeople: number
  imageURL: string;
  phone: string;
  email: string;
  lots?: Lot[];
  socialMedias?: SocialMedia[];
  eventsSpeakers?: Speaker[];
}
